### PR TITLE
shayan/Template dialog

### PR DESCRIPTION
**Pull request by shayan-binary-pr**

### CODE REVIEW

File: src/components/Views.jsx [hunk 1]
```diff
@@ -17,119 +16,177 @@ const ViewItem = ({
...
+    <div className="ciq-views__views__list__item" onClick={onClick}>
+        <span>{view.name}</span>
         <DeleteIcon onClick={remove} />
     </div>
 );
 
+const EmptyView = ({ onClick }) => (
+    <div className="ciq-views--empty">
+        <EmptyStateIcon />
+        <p>You have no save templates yet.</p>
```
Comment (morteza-binary) [inline src/components/Views.jsx]: @shayan-binary-pr the text should be translated.

**Comment by shayan-binary-2** [comment]
please close this PR as we create new one

**Comment by morteza-binary** [comment]
Duplicate of #932

**Comment by morteza-binary** [comment]
Duplicate of #932

**Pull request closed without merging**